pyscript.core/src/plugins/pyterminal.js

```diff
@@ -0,0 +1,110 @@
+// PyScript py-terminal plugin
+import { hooks } from "../core.js";
+
+// XXX TODO:
+// 1. these imports should be lazy?
+// 2. would be nice to automatically add xterm.css on demand
```

Comment:
we can grab its content and append it on `document.head` ... as long as the document is the right one (see workers).

Reply:
I was thinking of dynamically adding a `<link>` element.

> we can grab its content and append it on document.head

is there any advantage in doing this?

Reply:
the concept is the same, we add dynamically a tag or its content to the `document.head` and my "*we can grab its content*" was poorly worded: anyway we want to do so works, we still need to grab its content and append to the `document.head` either as link or style ... that's not what I was after, rather the fact in worker cases only we need that exsting or proxied `document` and add the style content in it.

Reply:
on the other hand, the link coudl be likely be added out of the box from main, no need to branch logic for workers cases only.